### PR TITLE
create a physical rack table

### PR DESCRIPTION
**Pull request by felipedf**
This create a new table physical_racks and provides a :physical_rack_id column to physical_servers table.

### CODE REVIEW

**Comment by felipedf** [comment]
@Fryguy

**Comment by agrare** [comment]
@miq-bot assign @Fryguy

File: db/migrate/20180116210559_add_physical_rack_to_physical_servers.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPhysicalRackToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :physical_rack_id, :bigint
```
Comment (agrare) [inline db/migrate/20180116210559_add_physical_rack_to_physical_servers.rb]: Don't think you need/want to do this in its own migration, adding a column to the new table in the above migration is fine /cc @Fryguy opinions?

**Comment by felipedf** [comment]
@agrare resolved

**Comment by Fryguy** [comment]
@felipedf I just realized this didn't include an ems_ref column, which is generally required if the table is to be filled in during inventory refresh.

**Pull request merged**